docs/topics/spiders.rst

```diff
@@ -283,7 +283,19 @@ Spider arguments are passed through the :command:`crawl` command using the
 
     scrapy crawl myspider -a category=electronics
 
-Spiders receive arguments in their constructors::
+Spider arguments are exposed as attributes::
```

Comment:
Exactly @eliasdorneles, from the current docs it seems like the only way to access spider input arguments is from the constructor arguments.

Maybe this example is better?

```
    import scrapy
    
    class MySpider(scrapy.Spider):
        name = 'myspider'

        def parse(self, response):
            products = response.css('.%s' % self.category)
````